cloudformation/xtdb_msk.yml

```diff
@@ -0,0 +1,161 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Parameters:
+  VpcId: 
+    Description: ID of the VPC to host the MSK cluster on
+    Type: AWS::EC2::VPC::Id
+
+  SecurityGroupId:
+    Description: Group ID of the security group to be used by the ECS nodes
+    Type: AWS::EC2::SecurityGroup::Id
+
+  MSKClusterName:
+    Type: String
+    Default: xtdb-cluster-kafka
+    Description: Enter the desired name of the MSK kafka cluster used as the XTDB Kafka Log
+
+  MSKVolumeSize:
+    Type: Number
+    Default: 100
+    Description: The size in GiB of the EBS volume for the data drive on each broker node of the kafka cluster.
+
+Resources:
+  PrivateSubnetMSKOne:
```

Comment:
For MSK you create the subnets within the MSK CF template, but for the ECS ALB you have them passed in - worth picking the same strategy for both? if so, any preferences?

Reply:
Hmmm.. It's a fair question. I think I'd prefer to keep them closer with the stuff that's actually using them, so I'm happy to create them within the ALB  template, will need to also expose them from there, too.

Reply:
Decided to go with the approach of creating all the subnets in the VPC template and passing references around :)